Source/Blazorise/Components/InputMask/InputMask.razor.cs

```diff
@@ -71,7 +71,7 @@ protected override async Task OnFirstAfterRenderAsync()
             Alias,
             InputFormat,
             OutputFormat,
-            Placeholder,
```

Comment:
Strange, I see this Placeholder being removed here, but I don't see a property reference being removed? Am I missing something?

Reply:
Or is it staying with a different purpose?

Reply:
MaskPlaceholder is now passed to JS lib.

Reply:
I see that. I meant that `Placeholder` was removed, but I guess it now serves a diffrent purpose, which is not the same as the maskplaceholder.

Reply:
Yes. Now, it is only applied directly to the input.